test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetSourcesTests.cs

```diff
@@ -475,6 +475,211 @@ private static void CreateXmlFile(string configFilePath, string nugetConfigStrin
             }
         }
 
+        [PlatformTheory(Platform.Windows)]
+        [InlineData(@"https://source.test", true)]
+        [InlineData(@"\\myserver\packages", false)]
+        public void Sources_WhenAddingSourceWithProtocolVersion_GotAdded(string source, bool shouldWriteProtocolVersion)
```

Comment:
Since the test uses `SimpleTestPathContext`, it can use a sub-directory as the source path for the second case, and therefore avoid hardcoding a Windows UNC path, and therefore no longer be limited to running only on Windows.

Reply:
I've decided to split those tests for clarity, similarly to what I did in NuGetSourcesCommandTest.cs.

When I tried using relative path for source, `IsLocal()` was returning `false`